index.bs

```diff
@@ -598,6 +598,8 @@ When this method is invoked, the user agent MUST execute the following algorithm
         to |authenticator|'s attachment modality, [=iteration/continue=].
     1. If {{AuthenticatorSelectionCriteria/requireResidentKey}} is set to |true| and the |authenticator| 
         is not capable of storing a [=Client-Side-Resident Credential Private Key=], [=iteration/continue=].
+    1. If {{AuthenticatorSelectionCriteria/matcherProtection}} is [=present|present=] and the
+        to |authenticator|'s matcher protection method is not included in that list, [=iteration/continue=].
```

Comment:
I don't see the matcher protection method in the getInfo operation. Does the client really know it, or does this information need to be sent down to the authenticator for it to do the filtering?